distribution/build.gradle

```diff
@@ -388,8 +390,11 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
         // module provided bin files
         with copySpec {
           eachFile { it.setMode(0755) }
-          if(testDistro == false) {
+          if (testDistro == false) {
+            from(defaultBinFiles)
+          } else {
             from(defaultBinFiles)
+              include 'x-pack-env', 'x-pack-security-env'
```

Comment:
Why is this explicit include necessary?

Reply:
We will need to call to `ConfigInitialNode` and to `EnrollNodeToCluster` from the startup script (either directly or by calling the bash script ) .Eitherway `elasticsearch-cli`. Either way, we need `x-pack-env` and `x-pack-security-env` as we do in all of our security CLI tools. My original change had us copying all bin files but [@mark-vieira commented](https://github.com/elastic/elasticsearch/pull/77632#discussion_r707753729) that we don't need all , so I'm just copying the two that we need.

Reply:
Hey @rjernst did that answer your question? Any additional concerns/thoughts? Thanks!

Reply:
It answered my question, thanks!